internal/spec/protection.go

```diff
@@ -6,10 +6,25 @@ import (
 
 // Protection rule of repositry branch
 type Protection struct {
-	// Require status checks for merge
-	Require []string `yaml:"require,omitempty"`
+	// Required status checks for merge
+	Checks []string `yaml:"checks,omitempty"`
 	// Strict update with target branch is requried
-	Strict bool `yaml:"strictUpdate"`
+	Strict bool `yaml:"strictUpdate,omitempty"`
+	// PullRequestReviewsEnforcement
+	Review struct {
+		// The pull request reviews enforcement of a protected branch.
+		Require bool `yaml:"require,omitempty"`
+		// Users who can dismiss review
+		Users []string `yaml:"users,omitempty"`
+		// Teams who can dismiss review
+		Teams []string `yaml:"teams,omitempty"`
```

Comment:
These fields look not clear -- `Review.Users` is not obvious to understand, I'd change it to `Review.Dismiss.Users` (with a nested struct)